src/main/java/Duke.java

```diff
@@ -1,10 +1,113 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final String TEXT_SPACER = "    ";
```

Comment:
I like how this constant name is all uppercase using underscore to separate the words. It will be easily changable in the future as we could easily just modify this variable instead of modifying at multiple locations.